samples/excel/85-preview-apis/data-types-entity-attribution.yaml

```
@@ -0,0 +1,235 @@
+order: 7
+id: excel-data-types-entity-attribution
+name: 'Data types: Entity value attribution properties'
+description: This sample shows how to set entity values with data provider attributions in the card layout.
```

Consider mentioning the featured properties first.
```suggestion
description: This sample shows how to set data provider attributions on entity values in the card layout.
```